### PR TITLE
Add video display to artist pages

### PR DESCRIPTION
**Pull request by venix12**
instead of #10761

### CODE REVIEW

**Comment by notbakaneko** [comment]
I'm guessing the video isn't necessarily going to be on youtube only?

**Comment by venix12** [comment]
Assumed so considering the newspost ones are hosted on assets.ppy.sh and we are talking the same videos here, @peppy may know better though.

**Comment by peppy** [comment]
It should always be self hosted on assets.

File: resources/views/artists/show.blade.php [hunk 1]
```diff
@@ -46,6 +46,14 @@
                     <h1>{{ $artist->name }}</h1>
 
                     {!! markdown($artist->description) !!}
+
+                    @if ($artist->video_url !== null)
+                        <div class="artist__video">
+                            <div class="u-embed-wide">
+                                <iframe src="{{ $artist->video_url }}" allowfullscreen></iframe>
+                            </div>
```
Comment (notbakaneko) [inline resources/views/artists/show.blade.php:54]: If they're always self-hosted then just use `video`; putting video url into iframe also clips the controls on Safari

**Review by notbakaneko — CHANGES_REQUESTED** [review]

**Review by notbakaneko — APPROVED** [review]

**Pull request merged**